src/Network/Receive.pm

```diff
@@ -6490,4 +6490,43 @@ sub GM_silence {
 	}
 }
 
+
+sub guild_storage_log {
+    my ($self, $args) = @_;
+
+    if ($args->{result} == 0 || $args->{result} == 1) {
+        my %action = (
+            0 => T('Get'),
+            1 => T('Put'),
+        );
+ 
+        my $storage_info = {
+            len => 83,
+            types => 'a4 v V C V a8 C v a8 Z24 Z24 C',
+            keys => [qw(ID nameID amount action upgrade uniqueID identified type_equip cards charName time attribute)],
```

Comment:
nameID => itemID

Reply:
##
# itemName($item)
#
# Resolve the name of an item. $item should be a hash with these keys:
# nameID  => integer index into %items_lut
# cards   => 8-byte binary data as sent by server
# upgrade => integer upgrade level

Reply:
:thinking: